src/megachatapi.h

```diff
@@ -62,37 +62,271 @@ class MegaChatCall
 public:
     enum
     {
-        CALL_STATUS_CONNECTING = 0,
-        CALL_STATUS_RINGING,
-        CALL_STATUS_CONNECTED,
-        CALL_STATUS_DISCONNECTED
+        CALL_STATUS_INITIAL = 0,        /// Initial state
```

Comment:
We should add some brief description of the class `MegaChatCall`, like how it's managed (notified via callbacks where MEGAchat keeps ownership, or getter like `getChatCalls()` returning a copy).
Also, a brief description of the status the call goes through, both for incoming and outgoing calls; and other high-level details that can be useful for developers using MEGAchat.

Reply:
Done!